app/src/components/Empty/empty.scss

```diff
@@ -0,0 +1,48 @@
+.empty-page {
+  height: 100vh;
+  font-size: 125%;
```

Comment:
Best to use `rem` here, as we do everywhere else

Reply:
Checked!